### DISCUSSION TITLE
Backup Phrase Not Being Received

### DISCUSSION THREAD

**Discussion by metasoul111** [Web3Auth/Web3Auth#1221] [Help]
Many users are having a major issue, on a random log in they are being prompted for a backup phrase. The UI tells them that the back phrase email was sent at specific time and date but the end user never gets the email. Then when they try to log into their account and they ask for a back phrase they don't have it and are locked out. The user then tries to go back to the original computer and the original browser they setup which should allow them to log in. But, it again asks then for the backphrase and they are stuck because they never received it. How do we fix this?

**Comment by chaitanyapotti** [comment]
Hello..we're using sendgrid service and these domains sometimes block our sender id.
we're in the process of migrating to aws ses to send ema
